### DISCUSSION TITLE
Fuel consumption is strange during takeoff

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#3733] [General]
Hi to all of you,

first of all, really nice work from you all developing people! 

I am wondering that the fuel burn after takeoff isn´t very high. For example after takeoff with a climb to round aboit 5000 feet, only used about 100 KG per engine?! Strange.... (without taxi!).

Is that a known issue? 

Greets Klaus

**Comment by hotshotp** [comment]
engine model is being worked on

**Comment by riveroalbert** [comment]
Absolutely, lets have patience. I've never seen it higher than 1200 kg/ hour in the sim, and I'm pretty sure it should be arround 2500 kg/h per engine at least

**Reply by CptBayer** [reply]
Hi folks,

after the last update (custom FBW), the fuel consumption is like a Cessna 172 :-p .... During cruise only between 600 and 700 kg/h per engine. During climb only 1000 kg ... 

From Takeoff to FL280, all together burned 600 kg .. hui ... hope you will fix this soon.... thanx in advance ..

greets Klaus

**Reply by andrejetcat** [reply]
ouuh, it would not good if my cessna 172 have a 600kg/hr fuel flow :D 
...
but as they already say... a new engine model is in development... so it wouldn´t make sense to fix it temporary. :) I also looking forward to the new engine model to have realistic fuel flows :)

**Reply by CptBayer** [reply]
Just for info any data:

flying time 2h after takeoff
GW round about 70 t
fuel used. 3 t :-p ...

Missing round about 1,5 to 2 t of fuel burn .. It looks like, that the fuel burn from update to update (custom FBW) will decrease .... I had during climb a consumption of 700 kg each engine per hour .... 

Greets klaus
